### ISSUE TITLE
node is backlevel

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#1108]
I'm seeing regular (> 1 in 10) failures on our official builds in the User Interface component

Typically the failure looks like:
```
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.6:install-node-and-npm (install node and npm) on project access-services-user-interface: Could not extract the Node archive: Could not extract archive: '/tmp/r/com/github/eirslett/node/10.12.0/node-10.12.0-linux-x64.tar.gz': EOFException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.6:install-node-and-npm (install node and npm) on project access-services-user-interface: Could not extract the Node archive: Could not extract archive: '/tmp/r/com/github/eirslett/node/10.12.0/node-10.12.0-linux-x64.tar.gz'
```

Whilst investigating I observe that node is being download by
maven-frontend-plugin  ( https://github.com/eirslett/frontend-maven-plugin ) 

The location should be ok, but further I notice it's downloading 10.12.0, whilst the current LTS is 10.16.0 & current version is 12.4.0 - backlevel code could expose more security issues.

Can we bring node up to current levels (either current stream or at least LTS) to mitigate this risk?
cc: @bogdan-sava @davidradl @grahamwallis 

I will raise a new issue if I figure out a pattern behind the build failures - this is open to track the version.

**Comment by cmgrote** [comment]
Agree it's good to try to keep it up-to-date, but just a heads-up that Node versions proceed at a very rapid pace (eg. from initial release of v8 to initial release of v10 was less than 1 year). I also believe it's the major version number (v10 in this case) that's considered LTS.

See: https://nodejs.org/en/about/releases/

**Comment by planetf1** [comment]
This will be addressed by #2390
